### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by tulikavijay**
# Fixes # 
Fixes : #566 
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
-  Add readme for using helpers and understanding examples

Note : I want to know if i am going in the right direction with this PR.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=683) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=683) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=683) it.</sub>

File: examples/helpers/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Using helper class to send emails
+You can use helper classes to customize the process of sending emails using SendGrid. Each process such as sending a mock email,
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
You can use helper classes to customize the process of sending emails using SendGrid. Each process (such as sending a mock email,
```

File: examples/helpers/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+building attachments, configuring settings, building personalizations,etc are made easy using helpers. All you need is a file with
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
building attachments, configuring settings, building personalizations, etc.) are made easy using helpers. All you need is a file with 
```

File: examples/helpers/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+## Using helper class to send emails
+You can use helper classes to customize the process of sending emails using SendGrid. Each process such as sending a mock email,
+building attachments, configuring settings, building personalizations,etc are made easy using helpers. All you need is a file with 
+all the classes imported and you can start sending emails!
+
+> Note :  you will need move this file to the root directory of this project to execute properly.
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
> Note: You will need move this file to the root directory of this project to execute properly.
```

File: examples/helpers/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+### Creating a simple email object and sending it
+The example [here](https://github.com/sendgrid/sendgrid-python/blob/0b683169b08d3a7c204107cd333be33053297e74/examples/helpers/mail_example.py#L9)
+defines minimum requirement to send an email.
+```
+ from_email = Email("test@example.com")
+ subject = "Hello World from the SendGrid Python Library"
+ to_email = Email("test@example.com")
+```
+you can use `Email` class to define a mail id.
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
You can use `Email` class to define a mail id.
```

File: examples/helpers/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+```
+content = Content("text/plain", "some text here")
+```
+The `Content` class takes mainly two parameters - MIME type and the actual content of the email, it then returns the JSON-ready representation of this Content.
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
The `Content` class takes mainly two parameters: MIME type and the actual content of the email, it then returns the JSON-ready representation of this content.
```

File: examples/helpers/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+```
+ mail = Mail(from_email, subject, to_email, content)
+```
+After adding the above we create a mail object using `Mail` class, it takes the following parameters - Email address to send from, Subject line of emails, Email address to send to,Content of the message.
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
After adding the above we create a mail object using `Mail` class, it takes the following parameters: email address to send from, subject line of emails, email address to send to, content of the message.
```

File: examples/helpers/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+for more information on parameters and usage, see [here](https://github.com/sendgrid/sendgrid-python/blob/master/sendgrid/helpers/mail/mail.py)
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
For more information on parameters and usage, see [here](https://github.com/sendgrid/sendgrid-python/blob/master/sendgrid/helpers/mail/mail.py)
```

File: examples/helpers/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+### Creating Personalizations
+
+To create personalizations, you need a dictionary to store all your email components. see example [here](https://github.com/sendgrid/sendgrid-python/blob/0b683169b08d3a7c204107cd333be33053297e74/examples/helpers/mail_example.py#L47)
```
Comment (misterdorm) [inline examples/helpers/README.md]:
```suggestion
To create personalizations, you need a dictionary to store all your email components. See example [here](https://github.com/sendgrid/sendgrid-python/blob/0b683169b08d3a7c204107cd333be33053297e74/examples/helpers/mail_example.py#L47)
```

**Comment by misterdorm** [comment]
You'll also need to agree to the CLA before this can be merged.  Click 'details' on the 'license/cla' check above to do so.  Thanks!

**Comment by tulikavijay** [comment]
This is an awesome feature! Thanks for the review!

**Comment by thinkingserious** [comment]
Hello @tulikavijay,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @tulikavijay,<br /><br/>Thanks again for the PR!<br /><br />                                         We noticed that you have earned 1 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold.<br /><br />                                         Also, please be sure that you have officially registered <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @tulikavijay,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 1 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request merged**